tests/pre_merge/deploy/test_inferencer.py

```diff
@@ -107,11 +107,12 @@ def test_openvino_inference(self, model_name: str, category: str = "shapes", pat
                 input_size=model_config.dataset.image_size,
                 onnx_path=export_path / "model.onnx",
                 export_path=export_path,
```

Comment:
I might be missing the point of having two paths here. Is there a specific reason?